pom.xml

```diff
@@ -269,6 +269,15 @@
                     <groupId>org.apache.felix</groupId>
                     <artifactId>maven-bundle-plugin</artifactId>
                     <version>3.3.0</version>
+                    <extensions>true</extensions>
+                    <executions>
+                        <execution>
+                            <id>baseline</id>
+                            <goals>
+                                <goal>baseline</goal>
+                            </goals>
+                        </execution>
+                    </executions>
```

Comment:
@davidjgonzalez what does this do?

Reply:
Makes sure the package-info.java's versions are correct based on the changes made to the public classes. if the package-info is wrong it will break the build and tell you what the correct version should be.

Reply:
gotcha. thanks

Reply:
Perfect example in this failing build :) 

![image](https://user-images.githubusercontent.com/1451868/36354603-01611a30-14a5-11e8-897e-86edf48e4ca1.png)